ui/src/main/webapp/wizards/ldap/stages/network-settings.js

```diff
@@ -74,7 +75,7 @@ const NetworkSettings = (props) => {
 
         <Port
           value={configs.port}
-          errorText={errors.port}
+          errorText={portError(configs.port) === undefined ? errors.port : portError(configs.port)}
```

Comment:
I would recommend computing the validation once and saving it into a variable instead of recomputing it.